### PR TITLE
Adding csv import service

### PR DESCRIPTION
**Pull request by cam156**

### CODE REVIEW

File: spec/services/metadata_application_profile_csv_importer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  let(:importer) { described_class.new(file) }
+
+  it 'parses a file' do
+    expect {
+      importer.import
+    }.to change(MetadataApplicationProfileField, :count).by(1)
+  end
+
+  context 'record exists' do
+    let(:field) { create :metadata_application_profile_field }
```
Comment (awead) [inline spec/services/metadata_application_profile_csv_importer_spec.rb]: Use `let!` here and you should be able to avoid the before call.

Comment (cam156) [reply spec/services/metadata_application_profile_csv_importer_spec.rb]: @awead rubocop does not enjoy let, so I started doing it this way.  Is rubocop now allowing let!

Comment (cam156) [reply spec/services/metadata_application_profile_csv_importer_spec.rb]: spec/services/metadata_application_profile_csv_importer_spec.rb:20:5: C: RSpec/LetSetup: Do not use let! for test setup.
    let!(:field) { create :metadata_application_profile_field }
    ^^^^^^^^^^^^

Comment (awead) [reply spec/services/metadata_application_profile_csv_importer_spec.rb]: weird!

**Pull request merged**